src/components/Callout/Callout.tsx

```diff
@@ -2,6 +2,8 @@ import * as React from 'react';
 import { ICalloutProps } from './Callout.Props';
 import { DirectionalHint } from '../../common/DirectionalHint';
 import { Layer } from '../../Layer';
+import { Button } from '../../Button';
```

Comment:
Lines 5 & 6 can be grouped into a single import, e.g. `import { Butotn, ButtonType } from '../../Button'`